src/tools/tidy/src/issues.txt

```diff
@@ -3618,6 +3615,7 @@ ui/recursion/issue-83150.rs
 ui/recursion/issue-86784.rs
 ui/recursion/issue-95134.rs
 ui/recursion_limit/issue-105700.rs
+ui/recursion_limit/issue-37311-type-length-limit/issue-37311.rs
```

Comment:
Would you please instead rename this and the associated .stderr files to
- tests/ui/recursion_limit/type-length-limit-issue-37311.rs
- tests/ui/recursion_limit/type-length-limit-issue-37311.stderr
- tests/ui/recursion_limit/type-length-limit-issue-37311.polonius.stderr

My reasoning is that there doesn't seem to be much merit in nesting these inside a folder that has the same name, and that this would allow eliminating these entries from this list.